rules/eslint/es6/walmart.js

```diff
@@ -19,6 +27,12 @@ module.exports = {
     // require method and property shorthand syntax for object literals
     "object-shorthand": 2,
     // suggest using of const declaration for variables that are never modified after declared
-    "prefer-const": 2
+    "prefer-const": 2,
+    // suggest using the spread operator instead of .apply()
+    "prefer-spread": 0,
+    // suggest using Reflect methods where applicable
+    "prefer-reflect": 0,
+    // disallow generator functions that do not have yield
+    "require-yield": 0
```

Comment:
I'd give this at least a `1`.  It is technically correct, and there may be cases where you'd want to do it, but you should have a good reason (which probably means an eslint line exclude).